atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolingContainer.java

```diff
@@ -151,6 +157,52 @@ protected void cleanupForDiscard(Client discardedResource) {
         super.cleanupForDiscard(discardedResource);
     }
 
+    public static class Builder{
+        private final String host;
+        private final int port;
+
+        // default values are provided
+        private int poolSize = CassandraKeyValueServiceConfig.DEFAULT.poolSize();
+        private String keyspace = CassandraKeyValueServiceConfig.DEFAULT.keyspace();
+        private boolean isSsl = CassandraKeyValueServiceConfig.DEFAULT.ssl();
+        private int socketTimeoutMillis = CassandraKeyValueServiceConfig.DEFAULT.socketTimeoutMillis();
+        private int socketQueryTimeoutMillis = CassandraKeyValueServiceConfig.DEFAULT.socketQueryTimeoutMillis();
```

Comment:
Is this the only place these defaults are used? If so, can we eliminate the DEFAULT class?